src/server.c

```diff
@@ -4620,6 +4640,51 @@ void bytesToHuman(char *s, unsigned long long n) {
     }
 }
 
+
+/* An array of time buckets, each representing a latency range,
+ * between 1 microsecond and roughly 1 second.
+ * Each bucket covers twice the previous bucket’s range.
+ * Empty buckets are not printed.
+ * Everything above 1sec is considered +Inf. */
+sds fillCumulativeDistributionLatencies(sds info, const char* histogram_name, struct hdr_histogram* histogram){
+    info = sdscatprintf(info, "latencyhist_%s:calls=%lld,histogram=[",
+        histogram_name, (long long) histogram->total_count);
+    struct hdr_iter iter;
+    hdr_iter_log_init(&iter, histogram, 2,2);
+    size_t bucket_pos = 0;
+    int64_t previous_count = 0;
+    while (hdr_iter_next(&iter))
+    {
```

Comment:
```suggestion
    while (hdr_iter_next(&iter)) {
```
{ goes on the previous line. The exception to this is if the if/while statement is multi-line